Dockerfile-dind-certs

```diff
@@ -0,0 +1,20 @@
+FROM python:2.7
```

Comment:
Since this is exclusively for testing purposes, I'd rather see this file moved to `tests/`